client/src/pages/CreatedConcert.js

```diff
@@ -16,60 +16,176 @@ import { ButtonFull } from "../components/buttons/ButtonFull";
+
+    setEdit(false);
+  };
 
   return (
     <>
       <GoBackHeader></GoBackHeader>
-      <Title size="big">{concert.artist}</Title>
-
-      <Divider />
-
-      <InfoContainer>
-        <DetailsItem>
-          <Icon src={Date} />
-          <p>{concert.date}</p>
-        </DetailsItem>
-
-        <DetailsItem>
-          <Icon src={Train} />
-          <p>{concert.arrival}</p>
-        </DetailsItem>
-
-        <DetailsItem>
-          <Icon src={Location} />
-          <p>
-            {concert.location}, {concert.city}
-          </p>
-        </DetailsItem>
-
-        <DetailsItem>
-          <Icon src={Euro} />
-          <p>{concert.price}</p>
-        </DetailsItem>
-      </InfoContainer>
-
-      <Divider />
-
-      <RowContainer>
-        <UserCard />
-        <DetailTextContainer>{concert.detailText}</DetailTextContainer>
-      </RowContainer>
-
-      <ButtonContainer>
-        <ButtonFull>Bearbeiten</ButtonFull>
-      </ButtonContainer>
+
+      {isLoading && "Loading..."}
+      {!isLoading && (
+        <>
+          {!edit ? (
+            <Title size="big">{concert.artist}</Title>
+          ) : (
+            <EditTitle
+              value={updatedArtist}
+              onChange={(event) => {
+                setUpdatedArtist(event.target.value);
+              }}
+            />
+          )}
+
+          <Divider />
+
+          <InfoContainer>
+            <DetailsItem>
+              <Icon src={Date} />
+              {!edit ? (
+                <p>{concert.date}</p>
+              ) : (
+                <EditInput
+                  value={updatedDate}
+                  type="date"
+                  onChange={(event) => {
+                    setUpdatedDate(event.target.value);
+                  }}
+                />
+              )}
+            </DetailsItem>
+
+            <DetailsItem>
+              <Icon src={Train} />
+              {!edit ? (
+                <p>{concert.arrival}</p>
+              ) : (
+                <EditArrivalDropdown
+                  value={updatedArrival}
+                  onChange={(event) => {
+                    setUpdatedArrival(event.target.value);
+                  }}
+                />
+              )}
+            </DetailsItem>
+
+            <DetailsItem>
+              <Icon src={Location} />
+              {!edit ? (
+                <p>
+                  {concert.location}, {concert.city}
+                </p>
+              ) : (
+                <>
+                  <EditInput
+                    value={updatedLocation}
+                    onChange={(event) => {
+                      setUpdatedLocation(event.target.value);
+                    }}
+                  />
+                  <EditInput
+                    value={updatedCity}
+                    onChange={(event) => {
+                      setUpdatedCity(event.target.value);
+                    }}
+                  />
+                </>
```

Comment:
you probably want a container that groups these two so you don't have city be so far away from the rest of the location:
![image](https://user-images.githubusercontent.com/48555130/80610830-1dbec800-8a3a-11ea-8845-fa6449581518.png)
I suggest using a parent flex element that manages remaining space accordingly. The first part of the address gets as much space it needs to display its content, the city gets the rest.